java/src/test/java/com/novoda/notils/date/SimpleDateFormatThreadSafeTest.java

```diff
@@ -0,0 +1,119 @@
+package com.novoda.notils.date;
+
+import org.junit.Test;
+
+import java.text.ParseException;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.TimeZone;
+
+import static org.junit.Assert.assertEquals;
+
+public class SimpleDateFormatThreadSafeTest {
+    String DATE_PATTERN = "yyyy-MM-dd'T'HH:mm:ss'Z'";
+
+    @Test
+    public void givenStringDateInISOFormat_WhenTimeZoneIsUTC_ThenUTCTimeIsTheSame() {
+        SimpleDateFormatThreadSafe formatter = givenFormatterForTimeZone(timeZoneUTC());
+        
+        Date date = parseTestDate("2016-01-01T10:00:00Z", formatter);
+
+        assertEquals(date, createUTCDate(2016, Calendar.JANUARY, 1, 10));
+    }
+
+    @Test
+    public void givenStringDateInISOFormat_WhenTimeZoneIsUTC_ThenUTCPlusOneTimeIsOneHourMore() {
+        // This test exercises British Summer Time (device) vs UTC (API)
```

Comment:
After talking with @ouchadam we found out the best thing to do was to the comment. Done in 9bce5bd